analyzer/src/funTest/assets/projects/synthetic/spdx/project/project.spdx.yml

```diff
@@ -15,12 +15,12 @@ externalDocumentRefs:
 - externalDocumentId: "DocumentRef-curl-7.70.0"
   checksum:
     algorithm: "SHA1"
-    checksumValue: "a2ab98f8e10bad892313ccc573b3820369551dcb"
```

Comment:
Why the tests were not failing if the checksums were incorrect ?

Reply:
Because the verification of checksum was only just added in this PR...

Reply:
Yes I found out. I was reading sequentially :)